app/views/content/landing/food-for-thought/_quotes.html.erb

```diff
@@ -45,4 +45,7 @@
       ]
     ) %>
   </section>
+<%= render CallsToAction::AdviserComponent.new(
+  text: "Get free advice from an experienced former teacher who can help you find school experience, learn what it's like to be in the classroom and answer all your questions about getting into teaching.",
```

Comment:
Just thought - we don't normally say `former teacher` - we normally say `an adviser with years of teaching experience`

(I appreciate that means the rest of the sentence might need tweaking too!)